day1OfJS/variable.js

```diff
@@ -0,0 +1,65 @@
+/*
+Create a variable.js file and declare variables and assign string, boolean, undefined and null data types
+
+let vr = "javascript";
+
+let tr = true;
+let fl = false; 
+
+let firstName
+console.log(firstName);
+
+let emptyValue = null;
+*/
+
+
+/*
+Declare four variables without assigning values
+
+let first
+let second
+let third
+let fouth
+*/
+
+
+/*
+Declare four variables with assigned values
+
+let first = 1;
+let second = 2;
+let third = 3;
+let fouth = 4;
+*/
+
+
+/*
+Declare variables to store your first name, last name, marital status, country and age in multiple lines
+
+let firstName
+let lastName 
+let maritalStatus 
+let country  
+let age
```

Comment:
This variable doesn't have your first name, last name, marital status, country and age

Reply:
Declared values - f165068